template/script.sh.erb

```diff
@@ -53,10 +58,7 @@ code-server \
     --auth="password" \
     --bind-addr="0.0.0.0:${port}" \
     --disable-telemetry \
-    <% if code_server_version < '4.5' %>
-    # code-server 4.5 will not accept this argument
-    --extra-extensions-dir="$CODE_SERVER_DATAROOT/extensions" \
-    <% end %>
+    #--extra-extensions-dir="$CODE_SERVER_DATAROOT/extensions" \
```

Comment:
I think I found what's throwing all of this off, and it's actually incredibly simple.

This comment in this commit throws this command line arg off. In the older commits, it's either the comment in the block or the fact that the block creates an empty line.

That is, when it's commented the command only has up to `--disable-telemetry` argument.

You can see this by adding a `set -x` before this command is issued.

You get logs like this and as you can see, we're not even passing the other arguments to the command.
```

Thu Mar  9 11:32:11 EST 2023: Started code-server

+ code-server --auth=password --bind-addr=0.0.0.0:51965 --disable-telemetry
[2023-03-09T16:32:12.822Z] info  code-server 4.8.3 977b853a1e162ab583aed64b1322d1515c57728c
[2023-03-09T16:32:12.824Z] info  Using user-data-dir ~/.local/share/code-server****
```

I got this to work correctly for 4.8. 
```
    <%- if code_server_version < '4.5' -%>
    --extra-extensions-dir="$CODE_SERVER_DATAROOT/extensions" \
    <%- else -%>
    --ignore-last-opened \
    <%- end -%>
```